src/features/timeline/LastPeriodAsk.js

```diff
@@ -0,0 +1,54 @@
+import React, { useState } from "react"
+import { Box, Button } from "@material-ui/core"
+import { useDispatch, useSelector } from "react-redux"
+import { upsertEntry } from "../entries"
+import LastDateInput from "../onboarding/Onboarding/LastDateInput"
+import { selectMainMensesTag } from "../settings"
+
+const LastPeriodAsk = () => {
+  const dispatch = useDispatch()
+  const [lastPeriod, setLastPeriod] = useState(new Date())
+  const mainMensesTag = useSelector(selectMainMensesTag)
+
+  const submitPeriodDate = async () => {
+    try {
+      await dispatch(
+        upsertEntry(lastPeriod, {
+          note: `#${mainMensesTag}`,
+        })
+      )
+    } catch (e) {
+      // install a toast library?
+      // https://react-hot-toast.com/
+      console.log(e)
+    }
+  }
+  return (
+    <>
+      <h2>Oh no</h2>
```

Comment:
**suggestion:** Move this into an AppMainToolbar, see profile page

Reply:
Upadted :)